packages/backend/src/execution/middleware.ts

```diff
@@ -39,3 +49,39 @@ export function middleware<TQuery = unknown, TContext = unknown>({
         transformQuery,
     };
 }
+
+export const aclMiddleware = middleware<AnyQuery, AnyContext>({
```

Comment:
Let's call it PermissionsMiddleware. Keep the list of names short: every name adds additional complexity.